src/components/com_tjcertificate/site/views/trainingrecord/tmpl/edit.php

```diff
@@ -102,6 +107,7 @@ class="p-5"
 		</div>
 	</div>
 	
+	<input type="hidden" id="assigned_user_id" value="<?php echo $this->item->user_id; ?>" />
```

Comment:
is this needed since you have already rendered the same field at line 55

Reply:
yes needed show the user selected, in ajax we are using this value to show selected